src/js/components/ServiceForm.js

```diff
@@ -28,13 +28,15 @@ class ServiceForm extends SchemaForm {
 
   handleFormChange() {
     // Handle the form change in the way service needs here.
+    this.props.onChange(...arguments);
     return;
   }
 
   getDataTriple() {
     let model = this.triggerTabFormSubmit();
     return {
-      model: SchemaFormUtil.processFormModel(model, this.multipleDefinition)
+      model: SchemaFormUtil.processFormModel(model, this.multipleDefinition),
+      isValid: this.validateForm()
```

Comment:
Let's please keep the return type of the `getDataTriple` as it is. Please introduce a new method, if you need a different return type.